.github/workflows/build-and-test.yml

```diff
@@ -31,10 +31,10 @@ jobs:
       - name: Build all projects without running tests
         run: ./gradlew --build-cache build -x test
 
-      - name: Upload spotbugs results
-        uses: github/codeql-action/upload-sarif@main
-        with:
-          sarif_file: service/build/reports/spotbugs/main.sarif
+#      - name: Upload spotbugs results
+#        uses: github/codeql-action/upload-sarif@main
+#        with:
+#          sarif_file: service/build/reports/spotbugs/main.sarif
```

Comment:
Should we completely remove this now?

Reply:
I forgot about this, I'll either remove it or fix it. Spotbugs does seem to catch things that sonarqube doesn't report. I seem to recall there was a problem getting it to use sarif output.